README.md

```diff
@@ -1 +1,39 @@
-# myportfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: setup and mobile version skeleton
+
+> This project is the first step in building a profesional developer portfolio. We start by seting it up and building the mobile version skeleton.
+
+![screenshot](images/Portfolio1.png)
+
+
+## Built With
+
+-HTML & CSS
+
+## Authors
+
+👤 **Emirjeta Veisllari**
+
+- GitHub: [@myelin0](https://github.com/myelin0)
+- Twitter: [@Amy_Albania](https://twitter.com/Amy_albania)
+- LinkedIn: [@Emirjeta Veisllari](www.linkedin.com/in/emirjeta-veisllari-aa5b71222)
```

Comment:
- [ ] [OPTIONAL] Please ensure all links in your README file are working. This is very important as it exudes professionalism.